parser_test.go

```diff
@@ -29,10 +29,6 @@ var messageTests = []struct {
 		IsNil: true,
 	},
 	{
-		Expect: ":asd  :",
-		IsNil:  true,
-	},
-	{
```

Comment:
This version of the parser is slightly lazier and doesn't count this as an invalid message... I'm not sure if it's worth fixing, but I'll take a look.